clib/dyn.mli

```diff
@@ -93,3 +93,6 @@ sig
 end
 
 module Make () : S
+
+(* forwarding variables used by plugins *)
+val forward_read_debug_cmd : (unit -> string) option ref
```

Comment:
This has nothing to do there. Dyn is a module to define dynamic, extensible types. This hook should live as high as possible in the dependency tree. Somewhere in `engine/` might be reasonable.

Reply:
I moved it to `engine/DebugHook.ml`.

Reply:
Still seems in the wrong place, who is the user of this forward declaration?

Also, you didn't delete the thing in `dyn.ml`.

Reply:
I fixed Dyn.  The variable is not in `tactic_debug.ml` because AFAIK `idetop.ml` can't statically link to files that under the `plugins` directory.  If there's a way to do that (and it's a good idea), let me know.

Reply:
Indeed the fact that this is a plugin makes it quite tricky, @ppedrot , what do you think? Should we add a registration mechanism ? This is already possible to do properly [for example it is what serapi does to load the seralization for plugins, but a bit cumbersome, so I dunno, I think the right place to put this would be `mltop` for now and don't indeed complicate so much with the registration.